Tutorials/Creating-a-Custom-Dashboard/index.md

```diff
@@ -3,12 +3,11 @@ product: "CMS"
 meta.Title: "Creating a Custom Dashboard"
 meta.Description: "A guide to creating a custom dashboard in Umbraco"
 versionFrom: 8.0.0
+verified-against: 9.0.0
 ---
 
 # Tutorial - Creating a Custom Dashboard
 
-_This tutorial was last tested on **Umbraco 8.11**_
-
```

Comment:
How come you're removing this?

Reply:
I thought it was no longer required since the article has been tested on the latest version and that was a disclamer sort of line that the article may or may not hold true for versions beyond 8.11 since its not tested. 😄 

I can add it back if required.

Reply:
Ah! It **is** a disclaimer to let people know which version this guide was last tested on.

Perhaps add it back and add the v9 version that is has been tested against as well?

Reply:
Done 💪